pkg/controller/controller_instance_test.go

```diff
@@ -278,7 +278,7 @@ func TestReconcileServiceInstanceNonExistentClusterServicePlan(t *testing.T) {
 	// check to make sure the only event sent indicated that the instance references a non-existent
 	// service plan
 	events := getRecordedEvents(testController)
-	expectedEvent := corev1.EventTypeWarning + " " + errorNonexistentClusterServicePlanReason + " References a non-existent ClusterServicePlan \"nothere\" on ClusterServiceClass (K8S: \"SCGUID\" ExternalName: \"test-serviceclass\") or there is more than one (found: 0)"
+	expectedEvent := corev1.EventTypeWarning + " " + errorNonexistentClusterServicePlanReason + " References a non-existent ClusterServicePlan (K8S: \"\" ExternalName: \"nothere\") on ClusterServiceClass (K8S: \"SCGUID\" ExternalName: \"test-serviceclass\") or there is more than one (found: 0)"
```

Comment:
I know this wasn't your code to begin with, but since it's getting long and hard to read, can you convert it to a `fmt.Sprintf` please?